spec/js/ClickCounter.spec.js

```diff
@@ -36,7 +30,26 @@ describe("ClickCounter is a class that counts clicks and turns them into donuts,
             expect(underTest.getAutoClickerCount()).toBe(1);
         })
     });
-
+    describe("FEATURE : The cost of each _Clicking Companion_ will go up with each purchase.", () => {
+        /*
+        > As a game designer, I want the game to become more difficult as the game progresses, so that users will continue to play the game.
+        - Increase the cost of the second _Clicking Companion_ by an additional ten percent to 110.
+        - Increase the cost of every additional _Clicking Companion_ by an additional ten percent.
+        */
+        it("Given a donut count of 210, purchasing 2 clickers should decrease the donut count to 0.", () => {
+            recordClicks(210, underTest);
+            underTest.purchaseAutoClicker();
+            underTest.purchaseAutoClicker();
+            expect(underTest.getDonutCount()).toBe(0);
+        });
+        it("Given a donut count of 331, purchasing 3 clickers should decrease the donut count to 0.", () => {
+            recordClicks(331, underTest);
+            underTest.purchaseAutoClicker();
+            underTest.purchaseAutoClicker();
+            underTest.purchaseAutoClicker();
+            expect(underTest.getDonutCount()).toBe(0);
+        });
```

Comment:
Consider removing the duplicate logic by refactoring it into a function on the global scope.